src/constants/welcomeModalProps.ts

```diff
@@ -0,0 +1,78 @@
+const newMainlineCommitsUser = [
+  {
+    img: "mainline_commits/carousel_1_05_06.png",
+    alt: "New UI preview image",
+    title: "Welcome to the new mainline commits experience!",
+    description:
+      "We designed this UI to make navigation faster and more intuitive. By default it displays failed tasks, but you can customize this view with filters.",
```

Comment:
"highlights" makes more sense to me because we are still displaying commits that passed.
```suggestion
      "We designed this UI to make navigation faster and more intuitive. By default it highlights failed tasks, but you can customize this view with filters.",
```